packages/compass-aggregations/src/modules/stage.spec.js

```diff
@@ -203,16 +206,9 @@ describe('Stage module', () => {
           }
         });
       });
+
       it('returns the stage string', () => {
-        expect(generateStageAsString(stage)).to.deep.equal(`{$bucket: {
-     groupBy: "$price",
-     boundaries: [ 0, 200, 400 ],
-     default: "Other",
-     output: {
-       "count": { $sum: 1 },
-       "titles" : { $push: "$title" }
-     }
-   }}`);
+        expect(generateStageAsString(stage)).to.deep.equal('{$bucket: {groupBy: \'$price\',boundaries: [0,200,400],\'default\': \'Other\',output: {count: { $sum: 1},titles: { $push: \'$title\'}}}}');
```

Comment:
Is there any way that we could keep the indentation? It makes things quite a bit more readable :)

Reply:
:+1:

Reply:
Actually there is no indentation in this case - that seems to be what mongodb-query-parser's toString does in this case for some reason.

Reply:
Sorry. stringify https://github.com/mongodb-js/query-parser/blob/406ed4ec774b9dea707c39b694e67d8d085545bc/lib/stringify.js#L121-L125

Reply:
Let me try toJSString..

Reply:
OK that was somehow harder and more work than the fix it self. Nested string escaping all the way down...